src/ast-parser.y

```diff
@@ -30,6 +32,7 @@
 
 #define RELOCATE_STACK(type, array, stack_base, old_byte_size, new_size)      \
   do {                                                                        \
+    WABT_STATIC_ASSERT(std::is_trivially_copyable<type>::value);              \
```

Comment:
Nit: I suppose you could just go ahead and use C++11 `static_assert` directly here. Presumably you can do them all in one pass but because `static_assert` also requires failure text (unless we go to C++14), that's not a trivial change. So it might make sense to go ahead and start using `static_assert` with text for new asserts.